src/plan/sticky/immix/gc_work.rs

```diff
@@ -1,16 +1,20 @@
 use crate::policy::gc_work::TraceKind;
 use crate::policy::gc_work::TRACE_KIND_TRANSITIVE_PIN;
 use crate::scheduler::gc_work::PlanProcessEdges;
+use crate::scheduler::gc_work::UnsupportedProcessEdges;
 use crate::{plan::generational::gc_work::GenNurseryProcessEdges, vm::VMBinding};
 
 use super::global::StickyImmix;
 
-pub struct StickyImmixNurseryGCWorkContext<VM: VMBinding>(std::marker::PhantomData<VM>);
-impl<VM: VMBinding> crate::scheduler::GCWorkContext for StickyImmixNurseryGCWorkContext<VM> {
+pub struct StickyImmixNurseryGCWorkContext<VM: VMBinding, const KIND: TraceKind>(
+    std::marker::PhantomData<VM>
+);
+
+impl<VM: VMBinding, const KIND: TraceKind> crate::scheduler::GCWorkContext for StickyImmixNurseryGCWorkContext<VM, KIND> {
     type VM = VM;
     type PlanType = StickyImmix<VM>;
-    type ProcessEdgesWorkType = GenNurseryProcessEdges<VM, Self::PlanType>;
-    type TPProcessEdges = GenNurseryProcessEdges<VM, Self::PlanType>;
+    type ProcessEdgesWorkType = GenNurseryProcessEdges<VM, Self::PlanType, KIND>;
+    type TPProcessEdges = UnsupportedProcessEdges<VM>;
```

Comment:
Will changing this break the Julia binding? @qinsoon @udesou

Reply:
I think this won't work if a binding uses transitive pinning. Can we use `TRACE_KIND_FAST` here? Will that work?

Reply:
That's what I had before in my fork, but then Eduardo mentioned that transitively pinning has unclear semantics for the modbuffer which I did not consider since ART does not do transitively pinning roots. I decided to make it explicit that transitively pinning is still unsupported instead of having a broken implementation in this PR.

Reply:
My question was that if this will completely break the Julia binding since it's changed to "unsupported" instead of "just working" tm since Julia disables copying for nursery GCs (i.e. sets `PREFER_COPY_ON_NURSERY_GC` to `false`).